.csscomb.json

```diff
@@ -0,0 +1,347 @@
+{
+  "always-semicolon": true,
+  "block-indent": 2,
+  "color-case": "lower",
+  "color-shorthand": true,
+  "eof-newline": true,
+  "leading-zero": false,
+  "quotes": "double",
+  "space-after-colon": 1,
+  "space-after-combinator": 1,
+  "space-after-selector-delimiter": 1,
+  "space-after-opening-brace": "\n",
+  "space-after-selector-delimiter": "\n",
+  "space-before-selector-delimiter": 0,
+  "space-before-colon": "",
+  "space-before-combinator": 1,
+  "space-before-opening-brace": 1,
+  "space-before-closing-brace": "\n",
+  "space-between-declarations": "\n ",
+  "vendor-prefix-align": true,
+  "verbose": true,
+  "strip-spaces": true,
+  "rules-delimiter": 1,
+  "sort-order": [
```

Comment:
@timurvafin, thanks for the idea. It appears that `scss-lint` is clever enough and also has the ability to disable any particular checks. Added corresponding configuration to `scss-lint` to enforce the same props order in https://github.com/fs/rails-base/commit/368b05a4e2f13383c3863104a7b633f6a97921b7.